tests/berlin/eip2930_access_list/test_tx_intrinsic_gas.py

```diff
@@ -0,0 +1,198 @@
+"""
+abstract: Tests [EIP-2930: Access list transaction](https://eips.ethereum.org/EIPS/eip-2930).
+Original test by Ori: https://github.com/ethereum/tests/blob/v15.0/src/GeneralStateTestsFiller/stEIP1559/intrinsicGen.js.
+"""
+
+from typing import List
+
+import pytest
+
+from ethereum_test_tools import (
+    AccessList,
+    Address,
+    Alloc,
+    Bytes,
+    Environment,
+    Hash,
+    StateTestFiller,
+    Transaction,
+    TransactionException,
+)
+from ethereum_test_tools import Opcodes as Op
+
+from .spec import (
+    Spec,
+    compute_access_list_intrinsic_gas,
+    compute_data_intrinsic_gas,
+    ref_spec_2930,
+)
+
+REFERENCE_SPEC_GIT_PATH = ref_spec_2930.git_path
+REFERENCE_SPEC_VERSION = ref_spec_2930.version
+
+pytestmark = pytest.mark.valid_from("Berlin")
+
+
+@pytest.mark.parametrize(
+    "data",
+    [
+        pytest.param(Bytes(b""), id="data_empty"),
+        pytest.param(Bytes(b"0x00"), id="data_1_zero_byte"),
+        pytest.param(Bytes(b"0x00000000"), id="data_4_zero_byte"),
+        pytest.param(Bytes(b"0xFF"), id="data_1_non_zero_byte"),
+        pytest.param(Bytes(b"0x00FF"), id="data_1_zero_byte_1_non_zero_byte"),
+        pytest.param(Bytes(b"0xFE00"), id="data_1_zero_byte_1_non_zero_byte_reversed"),
+        pytest.param(Bytes(b"0x0102030405060708090A0B0C0D0E0F10"), id="data_set_1"),
+        pytest.param(
+            Bytes(b"0x0102030405060708090A0B0C0D0E0F101112131415161718191a1b1c1d1e1f20"),
+            id="data_set_1",
+        ),
+        pytest.param(
+            Bytes(b"0x00010203040506000708090A0B0C0D0E0F10111200131415161718191a1b1c1d1e1f"),
+            id="data_set_2",
+        ),
+        pytest.param(
+            Bytes(b"0x01020304050607080910111213141516171819202122232425262728293031"),
+            id="data_set_31_bytes",
+        ),
+        pytest.param(
+            Bytes(b"0x000102030405060708090A0B0C0D0E0F101112131415161718191a1b1c1d1e1f"),
+            id="data_set_32_bytes",
+        ),
+        pytest.param(
+            Bytes(b"0x010203040506070809101112131415161718192021222324252627282930313233"),
+            id="data_set_33_bytes",
+        ),
+        pytest.param(
+            Bytes(b"0x000000000000000000000000000000000000000000000000000000000000000000"),
+            id="data_set_33_empty_bytes",
+        ),
+        pytest.param(
+            Bytes(
+                b"0x000000000000000000000000000000000000000000000000000000000000000000010203040506070809101112131415161718192021222324252627282930313233"
+            ),
+            id="data_set_66_bytes_half_zeros",
+        ),
+    ],
+)
+@pytest.mark.parametrize(
+    "access_list",
+    [
+        pytest.param([], id="access_list_empty"),
+        pytest.param(
+            [AccessList(address=Address(1), storage_keys=[])],
+            id="access_list_1_address_empty_keys",
+        ),
+        pytest.param(
+            [AccessList(address=Address(1), storage_keys=[Hash("0x60A7", left_padding=True)])],
```

Comment:
Could you try:
```suggestion
            [AccessList(address=1, storage_keys=[0x60A7])],
```
I think it should work out of the box.